content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -488,14 +478,13 @@ The folder structure for a _$JENKINS_HOME_ directory is as follows:
 JENKINS_HOME
  +- config.xml     (Jenkins root configuration file)
  +- *.xml          (other site-wide configuration files)
- +- identity.key   (RSA key pair that identifies an instance)
  +- secret.key     (deprecated key used for some plugins' secure operations)
  +- secret.key.not-so-secret  (used for validating _$JENKINS_HOME_ creation date)
  +- userContent    (files served under your https://server/userContent/)
  +- secrets        (root directory for the secret+key for credential decryption)
-     +- hudson.util.Secret   (used for encrypting some Jenkins data)
      +- master.key           (used for encrypting the hudson.util.Secret key)
-     +- InstanceIdentity.KEY (used to identity this instance)
```

Comment:
In my test installation, this file is now named org.jenkinsci.main.modules.instance_identity.InstanceIdentity.KEY. I'd rather have it renamed instead of being removed.